options/locale/locale_en-US.ini

```diff
@@ -2627,6 +2627,11 @@ diff.image.overlay = Overlay
 diff.has_escaped = This line has hidden Unicode characters
 diff.show_file_tree = Show file tree
 diff.hide_file_tree = Hide file tree
+diff.submodule_added = Submodule %s added at %s
+diff.submodule_added_link = Submodule <a href="%[1]s">%[2]s</a> added at <a href="%[1]s/commit/%[4]s">%[3]s</a>
```

Comment:
Can we move these links to the template rather than in the translation file?

Reply:
I will improve it by HTMLFormat